ui/components/app/tab-bar/tab-bar.js

```diff
@@ -16,10 +18,8 @@ const TabBar = (props) => {
           onClick={() => onSelect(key)}
         >
           <div className="tab-bar__tab__content">
-            <div className="tab-bar__tab__content__title">{content}</div>
-            <div className="tab-bar__tab__content__description">
-              {description}
-            </div>
+            <div className="tab-bar__tab__content__icon">{icon}</div>
+            <Typography variant={TYPOGRAPHY.H4}>{content}</Typography>
```

Comment:
@dragana8 This is the cause of the e2e test failure (but the code here is correct, the test needs to be updated).

Using the Typography component results in an `h4` element being rendered. The `send-eth.spec.js` test looks for a `div` element with the content `Advanced`. We just need to update that file to look for an `h4` tag instead. (line 190 of `send-eth.spec.js`